bindings/matrix-sdk-ffi/src/timeline/mod.rs

```diff
@@ -106,12 +108,19 @@ impl Timeline {
 
     async fn send_attachment(
         &self,
-        url: String,
+        filename: String,
         mime_type: Mime,
         attachment_config: AttachmentConfig,
+        caption: Option<String>,
+        formatted: Option<FormattedBody>,
         progress_watcher: Option<Box<dyn ProgressWatcher>>,
     ) -> Result<(), RoomError> {
-        let request = self.inner.send_attachment(url, mime_type, attachment_config);
+        let formatted: Option<RumaFormattedBody> = match formatted {
+            Some(p) => Some(RumaFormattedBody::from(p)),
+            None => None,
+        };
```

Comment:
nit: could this be something like

```suggestion
        let formatted = formatted.map(Into::into);
```

I think the compiler might be able to infer what the target type is, based on the use below.

Reply:
ah, did not know this one ( still halfway through the rust book :P )